content/cloud-docs/workspaces/creating.mdx

```diff
@@ -34,35 +34,43 @@ New workspaces can be created by teams with permission to manage workspaces. ([M
 
-1. [Navigate to the workspace list](/cloud-docs/workspaces/#listing-and-filtering-workspaces) and click the "+ New Workspace" button (near the top of the page).
+1. [Navigate to the workspace list](/cloud-docs/workspaces/#listing-and-filtering-workspaces) and click the **+ New Workspace** button (near the top of the page).
 
-1. On the first screen, choose your VCS provider (or choose "No VCS connection").
+1. Choose a VCS provider (or choose **No VCS connection**). The **Choose a repository** screen appears. This screen is skipped if you chose **No VCS connection**.
 
    -> **Note:** If you haven't added a VCS provider for your organization yet, choosing one here will prompt you to configure it. See [Connecting VCS Providers](/cloud-docs/vcs) for more information.
 
-1. On the second screen, choose a repository from the filterable list. This screen is skipped if you chose "No VCS connection".
+1. Choose a repository from the filterable list. The **Configure settings** screen appears.
 
    Some VCS providers limit the list's size. If a repository isn't listed, you can still choose it by name; scroll to the bottom of the list and enter its ID in the text field.
 
    -> **Note:** For some VCS providers, this list includes a drop-down menu for changing which account's repositories are shown. Other providers combine all available accounts into a single list.
 
-1. On the third screen, enter a name for the workspace. This defaults to the repository name, if applicable. The name must be unique within the organization, and can include letters, numbers, dashes (`-`), and underscores (`_`). See also our [advice for useful workspace names](/cloud-docs/workspaces/naming).
+1. Enter a name for the workspace. This defaults to the repository name, if applicable. The name must be unique within the organization, and can include letters, numbers, dashes (`-`), and underscores (`_`). See our [advice for useful workspace names](/cloud-docs/workspaces/naming).
 
-1. Optionally, click the "Advanced options" link on the third screen to configure some additional version control settings. (These settings not shown if you chose "No VCS connection".) For information about these settings, see:
+1. Optionally, click the **Advanced options** link on the third screen to configure some additional version control settings. (These settings not shown if you chose **No VCS connection**.) For information about these settings, see:
    - [Terraform Working Directory][workdir]
    - [Automatic Run Triggering][trigger]
    - [VCS branch][branch]
    - [Include submodules on clone][submodules]
 
-1. Confirm creation with the "Create workspace" button.
+1. Click the **Create workspace** button to confirm creation. The **Configure Terraform variables** screen appears. You can set values for undefined [Terraform variables](/cloud-docs/workspaces/variables#terraform-variables) within the configuration or click the **Go to workspace overview** button to skip this step.
```

Comment:
Should we mention something about variables with defaults? Any variables with default values won't show up on this screen.

Reply:
oo yes we should! let me push up a change for that now!